skrub/tests/test_table_vectorizer.py

```diff
@@ -756,3 +758,148 @@ def test_changing_types_int_float():
     table_vec.fit_transform(X_fit)
     res = table_vec.transform(X_transform)
     assert np.allclose(res, np.array([[1.0], [2.0], [3.3]]))
+
+
+def test_column_by_column():
+    X = _get_clean_dataframe()
+    table_vec_all_cols = TableVectorizer(
+        high_card_cat_transformer=GapEncoder(n_components=2, random_state=0),
+        cardinality_threshold=4,
+    )
+    table_vec_all_cols.fit(X)
+    for col in X.columns:
+        table_vec_one_col = TableVectorizer(
+            high_card_cat_transformer=GapEncoder(n_components=2, random_state=0),
+            cardinality_threshold=4,
+        )
+        table_vec_one_col.fit(X[[col]])
+        assert table_vec_one_col.get_feature_names_out() == [
+            feat
+            for feat in table_vec_all_cols.get_feature_names_out()
+            if feat.startswith(col)
+        ]
+
+
+@skip_if_no_parallel
+def test_parallelism() -> None:
+    # Test that parallelism works
+    X = _get_clean_dataframe()
+    # the gap encoder and the minhashencoder
+    # should be parallelized on all columns
+    # the one hot encoder should not be parallelized
+    for high_card_cat_transformer in [
+        lambda: GapEncoder(n_components=2, random_state=0),
+        OneHotEncoder,
+        lambda: MinHashEncoder(n_components=2),
+    ]:
+        table_vec_no_parallel = TableVectorizer(
+            high_card_cat_transformer=high_card_cat_transformer(),
+            cardinality_threshold=4,
+        )
+        X_trans = table_vec_no_parallel.fit_transform(X)
+        for joblib_backend in ["loky"]:
+            with joblib.parallel_backend(joblib_backend):
+                for n_jobs in [None, 2, -1]:
+                    table_vec = TableVectorizer(
+                        n_jobs=n_jobs,
+                        high_card_cat_transformer=high_card_cat_transformer(),
+                        cardinality_threshold=4,
+                    )
+                    X_trans_parallel = table_vec.fit_transform(X)
+                    assert_array_equal(X_trans, X_trans_parallel)
+                    assert table_vec.n_jobs == n_jobs
+                    # assert that all attributes are equal except for
+                    # the n_jobs attribute
+                    for attr in [
+                        "transformers_",
+                        "columns_",
+                        "types_",
+                        "imputed_columns_",
+                    ]:
+                        assert str(getattr(table_vec, attr)) == str(
```

Comment:
Yep, unless repeating the code, which might actually look better.
```py
assert table_vec.transformers_ == table_vec_no_parallel.transformers_
...
```
which might also make the error code more descriptive in case the assertion returns False

Reply:
This needs to be addressed :)

Reply:
I think it is, right?